src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/mercury/InteractiveMapsMercuryComponentObject.java

```diff
@@ -97,24 +103,24 @@ public void verifyFilterBoxWasExpanded() {
   }
 
   public void verifyMapModalIsNotVisible() {
-    driver.switchTo().activeElement();
-    Assertion.assertFalse(checkIfElementOnPage(mapDiv));
+//    driver.switchTo().activeElement();
```

Comment:
Please leave comment on why a line needs to be temporary commented out. Otherwise, remove completely.